external/js/cothority/src/byzcoin/arrange/config.ts

```diff
@@ -0,0 +1,57 @@
+import {Argument} from "../index";
+import {Darc as d} from "../../darc";
```

Comment:
I would not shorten the imports here - it's perhaps a little bit faster to write, but so much harder to read.

Reply:
I agree. Just followed an ongoing practice in the actual code though.